common/styles/components/_image_gallery.scss

```diff
@@ -52,3 +52,24 @@ $image-gallery-control-margin: 18px;
     max-height: 70vh;
   }
 }
+
+.basic-page {
+  .wobbly-edge {
```

Comment:
Does this hide all wobbly edges?

Reply:
It will hide any inside the body, yeah, but there aren't any others on the page.
This is really a stop gap until we look at the galleries properly.